src/iop/dither.c

```diff
@@ -94,10 +106,16 @@ const char *name()
   return _("dithering");
```

Comment:
If you'd like.  I didn't want to make the name overly long.

Reply:
We already have long(er) names like "rotation and perspective", "raw chromatic aberration"...

Reply:
https://github.com/darktable-org/darktable/blob/dc8b87c3cc75c58331b379b2ac23c33c02207a0c/src/develop/imageop.c#L409

There is something i stumbled across yesterday. At least is seems we don't use the same maximum size everywhere :-)

Reply:
How about "dither or posterize"?

Reply:
> How about "dither or posterize"?

Works for me. More in line with other names as we use verbs.

Reply:
> https://github.com/darktable-org/darktable/blob/dc8b87c3cc75c58331b379b2ac23c33c02207a0c/src/develop/imageop.c#L409
> 
> There is something i stumbled across yesterday. At least is seems we don't use the same maximum size everywhere :-)

It should be: `sizeof(dt_dev_operation_t)` instead of `20` indeed.

Reply:
Or just `sizeof(module->op)` as this is was is more common in the code. I'll fix this one.